Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8-alpine
+
+LABEL Name=Schemathesis
+
+WORKDIR /app
+ADD . /app
```

Comment:
It's usually better practice to use `COPY` instead of `ADD`, as `ADD` does fancy stuff, like resolve URLs, zips, and what not. It can have unexpected side effects

Reply:
I'd also recommend using relative paths to the `WORKDIR`, as it's easier to just change the `WORKDIR` and have all the paths adjusting automatically instead of having to update the same string in multiple places 🙂